### PR TITLE
Adding EST to message detail timestamp.

### PR DESCRIPTION
**Pull request by webinista**
- Closes department-of-veterans-affairs/messaging-fe#145

### CODE REVIEW

**Comment by U-DON** [comment]
I think the other place to add this for now is in `utils/helpers.js` in the `formattedDate` function so that it can show EST wherever we use the time.

File: src/js/messaging/utils/helpers.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ export function formattedDate(date, options = {}) {
   let dateString;
 
   if (momentDate.isSame(now, 'd')) {
-    dateString = momentDate.format('HH:mm');
+    dateString = `${momentDate.format('HH:mm')} <abbr title="Eastern Standard Time">EST</abbr>`;
```
Comment (U-DON) [inline src/js/messaging/utils/helpers.js]: I think this literally prints out the full string un-marked-up. To return it as markup, it should be:

`<span>{momentDate.format('HH:mm')} <abbr title="Eastern Standard Time">EST</abbr></span>`

And also import React in this module.

Comment (webinista) [inline src/js/messaging/utils/helpers.js]: Ok, I'll undo this and let you handle that. I am still using a mock API and have no way to test this on my end.

Comment (U-DON) [inline src/js/messaging/utils/helpers.js]: Sure, will do.

**Pull request merged**